specs/HubProtocol.md

```diff
@@ -348,8 +363,160 @@ World;0:;
 
 Note that the final frame still ends with the `;` terminator, and that since the body may contain `;`, newlines, etc., the length is specified in order to know exactly where the body ends.
 
+## MessagePack (MsgPack) encoding
+
+In the MsgPack Encoding of the SignalR Protocol, each Message is represented as a single MsgPack array containing items that correspond to properties of the given hub protocol message. The array items may be primitive values, arrays (e.g. method arguments) or objects (e.g. argument value). The first item in the array is the message type.
```

Comment:
Hmm, so this is basically a length prefix (conceptually). The array size is the length prefix surrogate. Is this how you'd usually represent an object in message pack? If you convert this to JSON would it be any array instead of an object? Is that weird or normal? Did we save space?